src/curated-corpus/components/AddProspectModal/AddProspectModal.tsx

```diff
@@ -70,7 +70,7 @@ export const AddProspectModal: React.FC<AddProspectModalProps> = (
     setIsManualSubmission,
     setIsRecommendation,
   } = props;
-
+  //TODO: @Herraj save button padding is off, cancel button outline is primary colour instead of neutral
   return (
```

Comment:
Good catch, I never noticed that! The lighter primary colour for the outline did stand out to me, but I completely forgot it was supposed to be grey instead. The place to fix that would be in the `SharedFormButtons` component (and perhaps even in the custom `Button` component that is used there), not on the Prospecting page. 

Given that it's out of scope for this ticket, would you be happy to file another one specifically for the button issue? I haven't made many changes to those files when I transitioned them over, but we should really take a closer look and probably get rid of the `hollow-outline` etc. types of buttons there. Perhaps we could even go back to default MUI buttons everywhere? We'd do a complete sweep.